contracts/solana/Anchor.toml

```diff
@@ -5,7 +5,7 @@ resolution = true
 skip-lint = false
 
 [programs.localnet]
-stork = "2TSL7JwuTu9co7yUizwuh8EVdd4d96vDo9JykQCw8SHi"
+stork = "3ZZfMdMb4Ar1X5Utk39nEa3TKmZSyUeirkUjbbuqBX4V"
```

Comment:
Can you revert this?

Reply:
yep. Theres a good chance we'll want to change this before we actually go live but lets leave it this way for now